### ISSUE TITLE
TN: Adjust starting balances

### ISSUE THREAD

**Issue by relativityboy** [steemit/tinman#109]
As a user of an account on the testnet that was created on Mainnet, my initial Testnet balance may be so low that my account has no utility. Make all accounts coming from Mainnet with Mainnet balances that have significant utility, still have useful-sized balances on Testnet *so that the holders of those accounts can use them to utilize the Testnet.

**AC**
- [x] in `# Results` describe what needs to be done so starting balances on the Testnet are such that the above is true.

**Comment by mvandeberg** [comment]
This might not be as straightforward. On main net my account, `vandeberg` has `1413848.769112 VESTS`. On test net the account only has `189.514421 VESTS`. Because of the vote dust threshold this means the account cannot vote.

**Comment by inertia186** [comment]
When we ran the full snapshot for the current testnet deploy, these are the numbers we use to determine proportionality:

```
total_vests: 392969144534573479
total_steem: 72337453156
total_vesting_steem: 194048993790
total_port_balance: 200000000000
total_port_vesting: 145689028885
total_port_liquid: 54309858023
vest_conversion_factor: 370739
steem_conversion_factor: 750784768519
```

##### Option A

Try a higher `total_port_balance` in `txgen.conf`.  This option has already been merged into master and will apply to the next testnet deploy.  We're going to double `total_port_balance` to `400000000000`.

Doubling `total_port_balance` would give us a new `vest_conversion_factor`:

```
total_vests: 392969144534573479
total_steem: 72337453156
total_vesting_steem: 194048993790
total_port_balance: 400000000000
total_port_vesting: 291378868601
total_port_liquid: 108620018307
vest_conversion_factor: 741480
steem_conversion_factor: 1501573715524
```

##### Option B

We could also adjust the `txgen.conf` option called `min_vesting_per_account`, which is currently set to `0.001 TESTS`.  We could set it to just over the dust threshold.

**Comment by inertia186** [comment]
##### Option C

Re-proportion `total_vests` when large accounts that get a haircut on `account_create_operation`.

Related to #112, certain accounts could not be created because their `vesting_shares` exceeded `STEEM_MAX_ACCOUNT_CREATION_FEE`.  Applying a maximum in `tinman` fixes #112 so those accounts can now be created but in doing so, changed the proportions so that `porter` ended up with a large unallocated balance.

For this option, when calculating `total_vests`, we should also apply `STEEM_MAX_ACCOUNT_CREATION_FEE` so that `total_vests` is adjusted.  This affects `vest_conversion_factor` of `tinman` as well, such that the `txgen` proportions become:

```
total_vests: 224511100097450018
total_steem: 71947828278
total_vesting_steem: 193061497510
total_port_balance: 200000000000
total_port_vesting: 145700875997
total_port_liquid: 54298043584
vest_conversion_factor: 648969
steem_conversion_factor: 754686345419
```

**Comment by inertia186** [comment]
**Result:** Recommend **Option C**
